desc/plotting.py

```diff
@@ -2306,11 +2306,22 @@ def plot_boozer_surface(
     N_booz = kwargs.pop("N_booz", 2 * eq.N)
     title_font_size = kwargs.pop("title_font_size", None)
 
+    # Todo: review this please
```

Comment:
What is the issue here?

Reply:
There shouldn't be any. I marked the objectives that used two grids with this comment in case these should be handled differently. I don't see any problem if `grid_compute` and `grid_plot` differ in whether they have an axis.

Reply:
OK yeah it seems fine to me. And I think @ddudt had some changes on another branch that got rid of the separate grids for plotting/computing?